packages/@ember/-internals/views/lib/system/event_dispatcher.js

```diff
@@ -234,12 +242,16 @@ export default EmberObject.extend({
     @private
```

Comment:
This change makes this method less "private" I think, at least "private" in the OOP-sense of being able to call it from outside (remains "private" in the semver-sense that it shouldn't be used in user-land). At least when `@ember/jquery` overrides the EventDispatcher with its jQuery-based implementation when Ember 4.0 will have dropped jQuery support, it has to properly support that method as it will be called from other places. Just to remind us of this... (see https://github.com/emberjs/ember-jquery/pull/31)

Reply:
Ya, unfortunately `@private` here is _mostly_ serving as a marker for semver version of privateness. `@ember/jquery` is "first party" and will be supported regardless...